features/enlinkd/adapters/collectors/ipnettomedia/src/main/java/org/opennms/netmgt/enlinkd/NodeDiscoveryIpNetToMedia.java

```diff
@@ -80,47 +80,31 @@ protected void runNodeDiscovery() {
         IpNetToMediaTableTracker ipNetToMediaTableTracker = new IpNetToMediaTableTracker() {
             public void processIpNetToMediaRow(final IpNetToMediaRow row) {
                 IpNetToMedia macep = row.getIpNetToMedia();
-                if (macep.getPhysAddress() != null && 
-                        macep.getNetAddress() != null && 
-                        macep.getSourceIfIndex() != null &&
-                        (macep.getIpNetToMediaType() == IpNetToMediaType.IPNETTOMEDIA_TYPE_DYNAMIC
-                        || macep.getIpNetToMediaType() == IpNetToMediaType.IPNETTOMEDIA_TYPE_STATIC)) {
-                    if (LOG.isDebugEnabled()) {
-                        LOG.debug("processIpNetToMediaRow: node [{}], mac address {} and ip {} mediatype {}. saving",
+
+                LOG.debug("processIpNetToMediaRow: node [{}], {}:{}:{}. ",
+                          getNodeId(),
+                          macep.getPhysAddress(),
+                          str(macep.getNetAddress()),
```

Comment:
This should be wrapped in a `LOG.isDebugEnabled() {` statement to avoid unnecessarily calling `str(macep.getNetAddress())`.

Reply:
commit c1872f7f58a3616e0b18f95044527c2a1beb6c6d (HEAD -> jira/HZN-1393)
Author: Antonio <rssntn67@yahoo.it>
Date:   Tue Jan 15 10:00:33 2019 +0100

    Added if control on LOG.debug to avoid calculation